ci/install-cross-rust.yml

```diff
@@ -37,7 +37,7 @@ steps:
       git checkout 718a19c
       git merge -m "No pseudo tty" pitkley/docker-no-pseudo-tty
       cargo install --force --path .
-    displayName: Instaling cross supprot
+    displayName: Install cross
```

Comment:
```suggestion
    displayName: Install cross-shell support
```

Reply:
This installs the build tool called "cross": https://github.com/rust-embedded/cross

Reply:
😆 That's hilarious ... then nvm